src/main/java/com/faforever/client/map/generator/MapGeneratorService.java

```diff
@@ -113,6 +113,8 @@ private void deleteGeneratedMaps() {
             .forEach(p -> noCatch(() -> FileUtils.deleteRecursively(p)));
       } catch (IOException e) {
         log.error("Could not list custom maps directory for deleting leftover generated maps.", e);
+      } catch (RuntimeException e) {
+        log.error("Could not delete generated map folder");
```

Comment:
Should we inform the user 🤔

Reply:
We could I just followed how it handled not being able to list the custom maps directory. 
The only real reason to inform I think would be so that they could delete it themselves if they wanted to but not sure if necessary or depending on how often it happens might just be annoying.